src/Analysis/Ast/Impl/Modules/Resolution/MainModuleResolution.cs

```diff
@@ -145,7 +147,9 @@ public MainModuleResolution(string root, IServiceContainer services)
                 var paths = await PythonLibraryPath.GetSearchPathsAsync(Configuration, fs, ps, cancellationToken);
                 cancellationToken.ThrowIfCancellationRequested();
                 return paths.MaybeEnumerate().Select(p => p.Path).ToArray();
-            } catch (InvalidOperationException) {
+            } catch (InvalidOperationException ex) {
+                _log?.Log(TraceEventType.Warning, "Exception getting search paths", ex);
+                _ui?.ShowMessageAsync($"An exception occured while getting the search paths; analysis may not be available. See the output panel for more information.", TraceEventType.Warning);
```

Comment:
And you can put it in resources as it is localizable thing (as opposed to tracing stuff).